lib/datastore/index.js

```diff
@@ -152,19 +179,146 @@ Datastore.int = function(value) {
 };
 
 /**
- * Helper function to get a Datastore Double object.
+ * Create a query for the specified kind.
  *
- * @param {number} value - The double value.
- * @return {object}
+ * @resource [Datastore Queries]{@link https://cloud.google.com/datastore/docs/concepts/queries}
+ *
+ * @see {module:datastore/query}
+ *
+ * @param {string=} namespace - Namespace.
+ * @param {string} kind - The kind to query.
+ * @return {module:datastore/query}
+ */
+Datastore.prototype.createQuery = function(namespace, kind) {
+  if (arguments.length === 1) {
+    kind = arrify(namespace);
+    namespace = this.namespace;
+  }
+
+  return new Query(namespace, arrify(kind));
+};
+
+/**
+ * Helper to create a Key object, scoped to the instance's namespace by default.
+ *
+ * You may also specify a configuration object to define a namespace and path.
+ *
+ * @param {...*=} options - Key path. To specify or override a namespace,
+ *     you must use an object here to explicitly state it.
+ * @param {object=} options - Configuration object.
+ * @param {...*=} options.path - Key path.
+ * @param {string=} options.namespace - Optional namespace.
+ * @return {Key} A newly created Key from the options given.
```

Comment:
That would be saying the method returns itself. There's not a great way to document this, because a `Key` object is part of the `Entity` class, which is hidden. I went with "Key" because it's at least suggesting it's a custom object type. Should I just say `object`?